app/lib/katello/capsule_content.rb

```diff
@@ -13,7 +13,26 @@ def lifecycle_environments(organization_id = nil)
       scope
     end
 
-    def pulp_repos(environments = lifecycle_environments, content_view = nil)
+    def pulp_server
+      uri = URI.parse(self.capsule.url)
+
+      Runcible::Instance.new(
+        :url      => "#{uri.scheme}://#{uri.host.downcase}",
+        :api_path => "/pulp/api/v2/",
+        :user     => "admin",
+        :timeout      => SETTINGS[:katello][:rest_client_timeout],
+        :open_timeout => SETTINGS[:katello][:rest_client_timeout],
+        :cert_auth => {:ssl_client_cert => ::Katello::Cert::Ca.ssl_client_cert,
+                       :ssl_client_key => ::Katello::Cert::Ca.ssl_client_key},
+        :logging  => {:logger     => ::Foreman::Logging.logger('katello/pulp_rest'),
+                      :exception  => true,
+                      :info       => true,
+                      :debug      => true }
+      )
+    end
```

Comment:
i think originally we were only going to add cert auth to the capsules, but ended up adding it to the main katello server too.  +1 for using cert auth everywhere